small_version.css

```diff
@@ -0,0 +1,549 @@
+@media screen and (max-width: 375px) {
```

Comment:
You need to set the media query to cover a minimum width of 375px and a max 768px as stated in [mobile milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=Minimum%20mobile%20screen,should%20be%20768px).

Right now your webpage breaks on screens greater than 375px 🤦‍♂️
<img width="483" alt="image" src="https://github.com/sibomana123/MV__Portfolio/assets/31897434/620be4d1-9581-4170-b1ba-804e7d0ad796">

To fix this;
Try to set the min-width 375px and max-width 768px in media query - [documentation](https://www.freecodecamp.org/news/media-queries-width-ranges/)

Reply:
Hi @bellom,

I was trying to make my portfolio to be responsive on all screens, could you check for me?

Reply:
Hey @sibomana123, 

I can see you made some changes - you only need one-liner of code.

kindly revert what you have done and try this - `@media screen and (min-width: 375px) and (max-width: 768px) {`

Reply:
Hi @bellom ,

Thank you for your vast and guidance to me ,  think  was trying to correct could you check for me ?

Reply:
@sibomana123 revert to this commit - [06f0c34](https://github.com/sibomana123/MV__Portfolio/pull/6/commits/06f0c34d00128efcae018a48b40d4e27a49d5ad2) before you add the hint above.

Reply:
@bellom  thanks for support